exchanges/gateio/gateio_wrapper.go

```diff
@@ -280,7 +284,31 @@ func (g *Gateio) CancelAllOrders(_ *exchange.OrderCancellation) (exchange.Cancel
 // GetOrderInfo returns information on a current open order
```

Comment:
This is more of a question for @thrasher- should this only be limited to open orders? There are no other IBotExchange funcs to get a specific orderid from an exchange... Unless I've missed it

Reply:
Great question!

I have also noticed that a lot more exchanges are moving towards requring a trading pair/market symbol on top of an order ID, GateIO for example do have a [getOrder](https://www.gate.io/api2#trade) end point that takes order id/trading pair and only returns information on that order that will save a lot of time and computations  

@thrasher- @shazbert any thoughts on making changes to GetOrderInfo paramaters at this stage?

Reply:
No changes to params in my opinion, we can add in specific params later and include an order or exchange account suite that needs to be updated with a syncer like orderbook, but we would need to expand wrapper funcs to include fetching of open/closed orders/positions. Out of scope. But a good issue raised.

Reply:
GetOrderInfo should return a single, specific order, irrespective of whether it's closed or open as we already have wrapper funcs for obtaining active/historic orders. This is useful for people who just want to use the exchange package rather than for bot functionality.

Moving forward though, the bot will maintain a list of orders which will allow for querying by a specific exchange or pair (like what Ryan mentioned above) but is out of scope for this PR.